RetrospectiveExtension.Frontend/dal/azureDevOpsWorkService.tsx

```diff
@@ -47,9 +51,10 @@ class WorkService {
    */
   public async getTeamFieldValues(teamId: string):
     Promise<TeamFieldValues> {
+    const projectId = await getProjectId();
     const teamContext = {
       project: '',
-      projectId: VSS.getWebContext().project.id,
+      projectId: projectId,
```

Comment:
on line 25 it's only `projectId`, here it's `projectId: projectId`

both of them work, but I think we should use the same style. I suggest to pick one of the styles and use it.

Reply:
Good catch! Fixed.